adapter/noop2/noop_test.go

```diff
@@ -79,82 +83,72 @@ func TestBasic(t *testing.T) {
 		t.Errorf("Got error %v, expecting success", err)
 	}
 
-	handler, err := builder.Build(nil, nil)
-	if err != nil {
-		t.Errorf("Got error %v, expecting success", err)
-	}
-
-	checkNothingHandler := handler.(checknothing.Handler)
-	result, caching, err := checkNothingHandler.HandleCheckNothing(nil)
-	if err != nil {
+	var handler adapter.Handler
+	if h, err := builder.Build(nil, nil); err != nil {
 		t.Errorf("Got error %v, expecting success", err)
+	} else {
+		handler = h
 	}
 
-	if !result {
-		t.Errorf("Got false, expecting true result")
+	var exptCheckRes = adapter.CheckResult{
+		Status:        rpc.Status{Code: int32(rpc.OK)},
```

Comment:
Didn't want to reference pkg/status. Let me know if I can, will make the change.